helix-term/src/commands/engine/components.rs

```diff
@@ -0,0 +1,2075 @@
+use std::{collections::HashMap, sync::Arc};
+
```

Comment:
Should this file (and steel.rs) be scoped under `commands/engine/steel`? This file still seems Steel specific